config/proto/probe.proto

```diff
@@ -39,28 +38,19 @@ message HermesProbeDef {
     CEPH = 2;
```

Comment:
Should we remove this line (at least for now) given that support for Ceph is on pause?

Reply:
Yes, ack.